cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -48,20 +48,6 @@
     - makedirs: True
 {% endif %}
 
-{% if grains['os_family'] == 'RedHat' %}
```

Comment:
Ha. I did have a bad merge (reverting a revert), but I think I fixed up everything. This delete is intentional.